lib/content.ex

```diff
@@ -95,4 +101,20 @@ defmodule Content do
       render.(conn, template, data: data)
     end
   end
+
+  @doc """
+  `wildcard_redirect/3` redirects a "/route.json" request to "/route"
+  such that a request to "/admin/profile.json" invokes "admin/profile"
+  `router` is the Phoenix Router for your app, e.g: `MyApp.Router`
+  see: https://github.com/dwyl/content#4-wildcard-routing 
+  """
+  def wildcard_redirect(conn, params, router) do
+    route = Enum.filter(router.__routes__(), fn r -> 
+      r.path == conn.request_path
+    end) |> List.first
```

Comment:
We could use the `Enum.find` function and the `&` anonymous function syntax.
But it works also well with `filter` and `List.first`
```suggestion
    route = Enum.find(router.__routes__(), &(&1.path == conn.request_path))
```

Reply:
Good plan @SimonLab 👍 
(_if they are functionally equivalent we might as well use the shorter one_)